build.gradle

```diff
@@ -66,14 +66,11 @@ publishing {
 			artifactId = "${project.archives_base_name}-${project.minecraft_major_version}"
 
 			from components.java
+			java.withSourcesJar()
 		}
 	}
-
-	// select the repositories you want to publish to
 	repositories {
-		// uncomment to publish to the local maven
-		// mavenLocal()
-		repositories {
+		if (project.hasProperty("siphalorMavenUser")) {
```

Comment:
this is copy/paste from one of you other projects so the rest of us can run these executions